operate/importer-8_5/src/main/java/io/camunda/operate/zeebeimport/processors/FlowNodeInstanceZeebeRecordProcessor.java

```diff
@@ -75,7 +75,7 @@ public FlowNodeInstanceZeebeRecordProcessor(
             flowNodeTreeCacheSize,
             flowNodeStore::findParentTreePathFor,
```

Comment:
Do we need `flowNodeStore` as a field, it is only used here?

Reply:
True, missed that. I will remove this.

Reply:
Already done on other PR which hit stable :)